day_5/questions.md

```diff
@@ -2,12 +2,25 @@
 
 1. What is a Hash, and how is it different from an Array in Ruby?
 
+A hash is similar to array in the fact that it stores data however a hash allows for non-numerical data to be mapped
+
 1. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
 
+pet_store = {turtles: 100, goldfish: 200, rabbits: 50}
+
 1. given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
 
+puts "#{IA}"
+
 1. With the same hash above, how would we get all the keys?  All the values?
 
+p states.keys
+p states.values
+
 1. What is another example of when we might use a hash?  In this case, why is a hash better than an array?
 
+A hash would be helpful when mapping out all of the different names and details of a project you are creating. It would allow you to easily sort and change all of the different items you may have in your directory
+
 1. What questions do you still have about hashes?
+
+My questions would be how your hashes affect the code you are writing and how it exactly works with the other commands and methods we have been learning
```

Comment:
Hashes have their own methods built in to ruby just like arrays, strings, and classes. Hashes are generally used for collections and always are organized into key/value pairs. Iterating over hashes is a common practice in programming in any language.